tracker/iou_tracker.py

```diff
@@ -19,6 +19,10 @@
 generates video panoptic segmentation with re-ordered identities based on IoU
 overlaps within consecutive frames.
 
+We recommend to use the 3 input channels as it allows for a wide range of
+instance IDs. The evaluator options can be configured with raw_panoptic_format:
+`three_channel_png` to export results in the recommended format.
```

Comment:
Just a note that please add a link to {kitti_step,motchallenge_step}_test_server_evaluation.md, once those files are ready.
Then users will have a better picture of what is going on.

Reply:
Ack.